### PR TITLE
patch: added get se config and set se config in fts3 #2227

### PR DESCRIPTION
**Pull request by DilaksunB**
patch: added get se config and set se config in fts3 #2227

### CODE REVIEW

**Comment by bari12** [comment]
I am not quite sure why github thinks that the entire file changed. It shows 1500+ lines as changed. Please check if your editor is adding some kind of new line-ending character to each line.

**Comment by DilaksunB** [comment]
I messed up with the line endings, should be fixed now.

File: lib/rucio/transfertool/fts3.py [hunk 1]
```diff
@@ -456,6 +456,86 @@ def list_se_status(self):
             return result.json()
         raise Exception('Could not retrieve transfer information: %s', result.content)
 
+    def get_se_config(self, storage_element):
+        """
+        Get the Json response for the configuration of a storage element.
+        returns a Json result for the configuration of a storage element.
```
Comment (bari12) [inline lib/rucio/transfertool/fts3.py]: :returns: ...

File: lib/rucio/transfertool/fts3.py [hunk 2]
```diff
@@ -456,6 +456,86 @@ def list_se_status(self):
...
+        if result and result.status_code == 200:
+            C = result.json()
+            config_se = C[storage_element]
+            return config_se
+        raise Exception('Could not get the configuration of %s , status code returned : %s', (storage_element, result.status_code if result else None))
+
+    def set_se_config(self, storage_element, inbound_max_active=None, outbound_max_active=None, inbound_max_throughput=None, outbound_max_throughput=None, staging=None):
+        """
+        Set the configuration for a storage element. Used for alleviating transfer failures due to timeout.
+        returns: JSON post response in case of success, otherwise raise Exception.
```
Comment (bari12) [inline lib/rucio/transfertool/fts3.py]: :returns:

**Review by bari12 — CHANGES_REQUESTED** [review]

**Review by bari12 — APPROVED** [review]

**Pull request merged**